### PR TITLE
Eng 10127 Remove catalog & project.xm & @procinfo from the test suite

### PR DESCRIPTION
**Pull request by pdshaw**
(Without creating discontinuity in the performance charts!)

### CODE REVIEW

File: tests/test_apps/voltkvqa/src/voltkvqa/procedures_withexport/PutMp.java [hunk 1]
```diff
@@ -36,12 +36,6 @@
 import org.voltdb.VoltType;
 import org.voltdb.DeprecatedProcedureAPIAccess;
 
-//@ProcInfo
-//(
-//  partitionInfo   = "store.key:0"
-//, singlePartition = true
-//)
-
```
Comment (akhanzode) [inline tests/test_apps/voltkvqa/src/voltkvqa/procedures_withexport/PutMp.java]: Isnt PutMP supposed to be MP?

Comment (pdshaw) [inline tests/test_apps/voltkvqa/src/voltkvqa/procedures_withexport/PutMp.java]: It was commented; just removed the dead code.
However maybe it was a mistake to comment it out in the first place? We can go over this.

File: tests/test_apps/voltkvqa/run.sh [hunk 2]
```diff
@@ -163,18 +169,18 @@ function http-benchmark() {
         --maxvaluesize=1024 \
         --usecompression=false \
         --warmup=15 \
-        --threads=250
+        --threads=35
 }
```
Comment (prosegay) [inline tests/test_apps/voltkvqa/run.sh]: double check that you really intended to change these values in this change set...

Comment (pdshaw) [inline tests/test_apps/voltkvqa/run.sh]: Yes, dropped intentionally. Was hitting connection exhaustion that left the HTTP interface essentially dead.

Will cross check to make sure apprunner setup reflects a similar reduction.

**Pull request merged**